Average of digits

vipulkmr02 opened the pull request:
### Why:

<!-- If this PR is linked to any issue -->
<!-- please change #ISSUE_NUMBER to the respective issue number -->
<!-- For example, Closes codinasion/codinasion#2 -->
Closes codinasion/codinasion#4021 

### What's being changed:

vipulkmr02 committed:
final commit

vipulkmr02 committed:
removed some lines of code

hi-Kartik2004 committed:
Merge branch 'master' into averageOfDigits

anandfresh commented on program/find-average-of-digits-of-a-number/FindAverageOfDigitsOfANumberByRecursion.kt:
```diff
@@ -0,0 +1,13 @@
+import java.util.Scanner
```
@Coder-12-ux rename the file , you have used the same file name for both recursion and non-recursion

anandfresh committed:
Merge branch 'master' into averageOfDigits

hi-Kartik2004 commented:
Hi @Coder-12-ux, Firstly thanks for making a PR (pull request) I see that you have linked only one issue out of the two issues solved in this pull request, can you link both issues by editing your first / main comment with 

closes `#issuenumber1`
closes `#issuenumber2`

harshraj8843 committed:
Merge branch 'master' into averageOfDigits

harshraj8843 committed:
Delete FindAverageOfDigitsOfANumberByRecursion.kt

Issue #3304 has already closed

File 👉🏻 https://github.com/codinasion/program/blob/master/program/find-average-of-digits-of-a-number/FindAverageOfDigitsOfANumber.kt

harshraj8843 reviewed (approved):
Thanks for contributing 🤗

hi-Kartik2004 committed:
Merge branch 'master' into averageOfDigits
